modules/openapi-generator/src/main/java/org/openapitools/codegen/utils/ModelUtils.java

```diff
@@ -1711,9 +1699,16 @@ private static void setNumericValidations(Schema schema, BigDecimal multipleOf,
         }
     }
 
+    private static void logWarnMessagesForIneffectiveValidations(Set<String> setValidations, Schema schema, Set<String> effectiveValidations) {
+        setValidations.removeAll(effectiveValidations);
+        setValidations.stream().forEach(validation -> {
+            LOGGER.warn("Validation '" + validation + "' has no effect on schema. Ignoring!");
```

Comment:
How about adding schema's type here?
I see this message in our logs:
```
16:24:40.978 [TestNG-test=Surefire test-1] WARN org.openapitools.codegen.utils.ModelUtils - Validation 'minItems' has no effect on schema. Ignoring!
```
It would be more helpful if we can see that minItems didn't apply if the schema was type object schema.